fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -413,7 +413,7 @@ public class Config extends ConfigBase {
      * some hang up problems in java.net.SocketInputStream.socketRead0
      */
     @ConfField
-    public static int thrift_client_timeout_ms = 0;
```

Comment:
I  don't think changing a configuration name is appropriate. Perhaps someone has already used that configuration in a production environment. Better reserve the old one for compatibility.

Reply:
done